serializer.rst

```diff
@@ -196,10 +196,10 @@ In addition to the ``@Groups`` annotation, the Serializer component also
 supports Yaml or XML files. These files are automatically loaded when being
 stored in one of the following locations:
 
-* The ``serialization.yml`` or ``serialization.xml`` file in
+* The ``serialization.yaml`` or ``serialization.xml`` file in
   the ``Resources/config/`` directory of a bundle;
-* All ``*.yml`` and ``*.xml`` files in the ``Resources/config/serialization/``
-  directory of a bundle.
+* All ``*.yaml`` and ``*.xml`` files in the ``Resources/config/serialization/``
+  directory of a bundle and in the ``config/serializer/`` directory of your project.
```

Comment:
Should we mention the `config/serializer/` directory if your project first *before* talking about `Resources/config` in your bundle?